src/vivarium/manager.py

```diff
@@ -26,25 +29,57 @@ class Manager:
     ##############
 
     @property
-    def configuration_defaults(self) -> Dict[str, Any]:
+    @abstractmethod
```

Comment:
So is there a reason we wouldn't want to have the same name parsing as what is in component so we know any NAMEManager has the property name_manager?

Reply:
I just didn't want to make this more complicated. We could definitely introduce that.